packages/moonstone/GridListImageItem/GridListImageItem.js

```diff
@@ -121,7 +131,7 @@ const GridListImageItemBase = kind({
 		publicClassNames: ['gridListImageItem', 'icon', 'image', 'selected', 'caption', 'subCaption']
 	},
 
-	render: ({css, selectionOverlay, ...rest}) => {
+	render: ({css, 'data-webos-voice-intent': voiceIntent, selectionOverlay, ...rest}) => {
```

Comment:
It is not necessary to destructure this since it's just passed in with `{...rest}` below.